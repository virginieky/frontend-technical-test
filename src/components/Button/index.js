import styled from 'styled-components';

const Button = styled.button`
  display: flex;
  align-items: center;
  width: fit-content;
  height: 60px;
  margin: auto;
  padding: 0 10px;
  background: transparent;
  border: 0;
  color: #ff6e14;
  outline: 0;
  span {
    margin-left: 10px;
  }
`;

export default Button;
