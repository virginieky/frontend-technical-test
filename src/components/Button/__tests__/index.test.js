import React from 'react';
import { render } from '@testing-library/react';
import Button from '../index';

describe('components | Button', () => {
  it('should match the snapshot', () => {
    const { asFragment } = render(<Button />);

    expect(asFragment()).toMatchSnapshot();
  });
});
