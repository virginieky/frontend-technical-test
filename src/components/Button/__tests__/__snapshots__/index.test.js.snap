// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components | Button should match the snapshot 1`] = `
<DocumentFragment>
  <button
    class="Button-sc-1bt4yv5-0 jGLend"
  />
</DocumentFragment>
`;
